packages/react-scripts/config/paths.js

```diff
@@ -52,7 +88,10 @@ module.exports = {
   testsSetup: resolveApp('src/setupTests.js'),
   appNodeModules: resolveApp('node_modules'),
   ownNodeModules: resolveApp('node_modules'),
-  nodePaths: nodePaths
+  nodePaths: nodePaths,
+  publicUrl: getPublicUrl(resolveApp('package.json')),
```

Comment:
I went ahead and just removed the third variable as it was unnecessary. `publicUrl` and `servedPath` seem straightforward to me (now that `servedUrl` is gone).